tests/compile/piecewise/piecewise_compilation_config.json

```diff
@@ -1,4 +1,4 @@
 {
     "use_cudagraph": true,
-    "non_cudagraph_ops": ["silly.attention"]
+    "non_cudagraph_ops": ["vllm.toy_attention"]
```

Comment:
I kind of liked the old name

Reply:
because of the pytorch bug, I can only define the library in `vllm/utils.py` :( and I don't want to leave a `silly_lib = Library("silly", "FRAGMENT")` in the main code path.

Reply:
kept the old name now!